docs/projects/digital_bone_box/about.md

```diff
@@ -17,7 +17,7 @@ The app will categorize the bones into modules such as the bony pelvis, lower li
 
 - **Source Code:** -[https://github.com/oss-slu/DigitalBoneBox](https://github.com/oss-slu/DigitalBoneBox) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/DigitalBoneBox)
 - **Client** Dr. Brian Elliot - Program Director - Clinical Health Sciences
-- **Current Tech Lead:** Santosh Iragavarapu [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/jacksayshi) 
```

Comment:
Make sure you move Santosh into the Developers section and give him a `(alumni, prior tech lead)` label. You can see how we do that handoff in projects in like https://oss-slu.github.io/projects/esp/about and https://oss-slu.github.io/projects/dads/about